sources/AutoFilerTab.cpp

```diff
@@ -39,8 +46,8 @@ AutoFilerTab::AutoFilerTab()
 	BMessage panelMsg(MSG_FOLDER_CHOSEN);
 	fFilePanel = new BFilePanel(B_OPEN_PANEL, NULL, NULL,
 		B_DIRECTORY_NODE, true, &panelMsg, fRefFilter);
-	fFilePanel->Window()->SetTitle("AutoFiler: Add folders");
-	fFilePanel->SetButtonLabel(B_DEFAULT_BUTTON, "Add");
+	fFilePanel->Window()->SetTitle(B_TRANSLATE("AutoFiler: Add folders"));
+	fFilePanel->SetButtonLabel(B_DEFAULT_BUTTON, B_TRANSLATE("Add"));
```

Comment:
That German example was just to illustrate that it's the nature of buttons, that their practical width is limited.
Generally it's a good idea to provide the translators with the info where a string appears. More so, if it's just one word compared to sentences and it's maybe even used in several places in the app. An "Add" at one place may be differently translated when in another.

Maybe the code appears a bit less cluttered when the comment string is put in a new line.